Sources/Hummingbird/Application.swift

```diff
@@ -20,6 +20,12 @@ import NIOPosix
 import ServiceLifecycle
 import UnixSignals
 
+#if os(iOS) && canImport(NIOTransportServices)
```

Comment:
I've always used `canImport(Network)`. You don't need the platform check. 

CanImport on SwiftPM modules is decidedly flakey.

Reply:
Actually looking at the code it is only iOS that uses `NIOTSEventLoopGroup` by default.
This would do
```
#if os(iOS)
import NIOTransportServices
#endif
```

You'll need to run swift-format on the file after as swift-format orders all conditional imports after non-conditional ones

Reply:
On it, thank you.

Reply:
Adopted in [Simplify back to just #if os(iOS). Checked with a swift-format run …](https://github.com/hummingbird-project/hummingbird/pull/659/commits/c1d328c87597beb2fb16a264f0d8614940b2f432).